services/vaadin-directory/vaadin-directory-version.service.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const { BaseVaadinDirectoryService } = require('./vaadin-directory-base')
+
+module.exports = class VaadinDirectoryVersion extends BaseVaadinDirectoryService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'vaadin-directory',
+      pattern: ':which(v|version)/:packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Vaadin Directory',
+        pattern: 'v/:packageName',
+        namedParams: { packageName: 'vaadinvaadin-grid' },
+        staticPreview: this.render({ version: 'v5.3.0-alpha4' }),
+        keywords: ['vaadin-directory', 'version', 'latest version'],
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'vaadin directory' }
+  }
+
+  static render({ version }) {
+    return { message: version, color: '#00b4f0' }
```

Comment:
I know the color is consistent with the current/legacy implementation, but out of curiosity do you know why we use a static `#00b4f0` color here instead of the `version` colorFormatter used for other version badges?

Reply:
I just reproduced the existing behaviour, but I don't think there's a clear reason for it. Having just done a bit of reading, Vaadin Directory does seem to at least assume SemVer if not enforce it, so I think its more useful to standardize on `renderVersionBadge()`